docs/compass/08-01-basic-flow-tutorial.md

```diff
@@ -31,45 +31,44 @@ For simplicity reasons, use the available Order Service as the sample external A
 
 ### Register your Application in the Compass UI
 
-1. Select **Back to Namespaces** in the top-left corner of the page, and go to the **Compass** tab in the left navigation panel. It will navigate you to the Compass UI. Select a tenant you want to work on from the drop-down list on the top navigation panel. For the purpose of this tutorial, select the `default` tenant. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial. Make sure that your Runtime is assigned to the `DEFAULT` scenario.
+1. Open a separate tab in your browser and go to  `https://compass.{CLUSTER_DOMAIN}` that will navigate you to the Compass UI. Select a tenant you want to work on from the drop-down list on the top navigation panel. For the purpose of this tutorial, select the `default` tenant. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial. Make sure that your Runtime is assigned to the `DEFAULT` scenario.
```

Comment:
```suggestion
1. Open a separate tab in your browser and go to `https://compass.{CLUSTER_DOMAIN}`. It will navigate you to the Compass UI. Select a tenant you want to work on from the drop-down list on the top navigation panel. For the purpose of this tutorial, select the `default` tenant. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial. Make sure that your Runtime is assigned to the `DEFAULT` scenario.
```